aws/prometheus-data/main.tf

```diff
@@ -1,20 +1,30 @@
-data "aws_caller_identity" "this" {}
+data "aws_region" "this" {}
 
-data "aws_s3_bucket_object" "prometheus" {
```

Comment:
We no longer need to use the bucket because there is an `aws_prometheus_workspace` data source in the newer Terraform provider.